docs/screenplay_elements/Web/Miscellaneous/utils.md

```diff
@@ -35,13 +36,13 @@ const getSubLocator = async (locator: Locator, subLocator?: Locator, text?: stri
 
 ```typescript
 const subLocatorLookup = async ({
-    page, locator, timeout, subSelector, state = 'visible',
-}: { page: Page; locator: Locator; timeout?: number; subSelector?: SubSelector, state?: SelectorOptionsState }): Promise<Locator>;
+    base, locator, timeout, subSelector, state = 'visible',
+}: { base: Page; locator: Locator; timeout?: number; subSelector?: SubSelector, state?: SelectorOptionsState }): Promise<Locator>;
```

Comment:
is the type for base correct?

Reply:
Since we implemented the framelocator support. 
const base = frameTree === undefined || frameTree.length === 0 ? page : recursiveFrameLookup(page, frameTree);